src/main/steps/applicant1/dob/content.ts

```diff
@@ -0,0 +1,90 @@
+import { CaseDate } from '../../../app/case/case';
+import { TranslationFn } from '../../../app/controller/GetController';
+import { FormContent } from '../../../app/form/Form';
+import { covertToDateObject } from '../../../app/form/parser';
+import { areDateFieldsFilledIn, isDateInputInvalid, isFutureDate } from '../../../app/form/validation';
+
+export const en = (): Record<string, unknown> => ({
+  section: 'Primary applicant',
+  title: "What's your date of birth?",
+  hint: 'For example, 28 6 1997',
+  errors: {
+    dateOfBirth: {
+      required: 'You have not entered a date. Enter a date to continue.',
+      invalidDate: 'You have entered an invalid date. Enter the date using the following format: 28 6 1997.',
+      invalidYear: 'You have entered the year in an invalid format. Enter the whole year, for example 2002.',
+      invalidDateInFuture:
+        'You have entered a date that is in the future. Enter a date that is in the past before continuing.',
+      invalidDateTooFarInPast: 'You have entered a year which is too far in the past.',
```

Comment:
are these messages provided by UCD?
can you point me to the place where we can see these error messages?

Reply:
This date field was a 'lift and shift' from nf-div, I assumed error messages were standardised for date fields. Please let me know if there is an inconsistency here

Reply:
Nope error messages and all the content should come from UCD
In your case, please refer the https://tools.hmcts.net/jira/browse/ADOP-149 for error messages in different scenarios